lib/netsuite/actions/add.rb

```diff
@@ -1,7 +1,9 @@
 # https://system.netsuite.com/help/helpcenter/en_US/Output/Help/SuiteCloudCustomizationScriptingWebServices/SuiteTalkWebServices/add.html
+require_relative 'abstract_action'
```

Comment:
I don't think we need these, I'm pretty sure we can `autoload` them in `lib/netsuite`

Reply:
Ahhh I didn't see that I could do that. Much cleaner.